src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -78,6 +82,28 @@ export class MapproxySeed {
     }
   }
 
+  public addTimeMinuteBuffer(dataTimeStr: string): string {
+    const secondsInMin = 60;
+    const factor = 2;
+    const timeBufferMinute = (this.gracefulReloadMaxSeconds / secondsInMin) * factor;
+    const origDateTime = new Date(dataTimeStr);
+    const minutes = origDateTime.getMinutes() + timeBufferMinute;
+    const newDateTime = new Date(origDateTime.setMinutes(minutes));
+
+    const nowUtc = Date.UTC(
```

Comment:
Consider one of the following:
* Pass UTC time from overseer (and then just add the minutes)
* Add a new method to return UTC time from local time in utils like here https://github.com/MapColonies/mc-utils/blob/master/src/dateTime/getUTCDate.ts#L1